jsapp/js/components/common/tabs.module.scss

```diff
@@ -7,30 +7,25 @@
   margin: 0 sizes.$x40;
   display: flex;
   flex-direction: row;
-  border-bottom: 1px solid colors.$kobo-gray-85;
+  border-bottom: 2px solid colors.$kobo-gray-85;
 }
 
 .tab {
   background: transparent;
   cursor: pointer;
   font-size: sizes.$x18;
-  color: colors.$kobo-gray-40;
-  font-weight: 500;
+  color: colors.$kobo-gray-55;
+  font-weight: 600;
   padding: sizes.$x14 sizes.$x30 sizes.$x10;
-  margin-bottom: -1px;
+  margin-bottom: -2px;
```

Comment:
Looks good! Sorry, one last thing I missed. Could you use `scss/sizes` for these pixel values? You can go ahead and merge once that's finished.